isso/js/app/templates/comment.pug

```diff
@@ -1,21 +1,21 @@
-div(class='isso-comment' id='isso-#{comment.id}')
+div(class='isso-comment' id='isso-' + comment.id)
     if conf.gravatar
         div(class='avatar')
-            img(src='#{comment.gravatar_image}')
+            img(src=comment.gravatar_image)
     if conf.avatar
         div(class='avatar')
-            svg(data-hash='#{comment.hash}')
+            svg(data-hash=comment.hash)
     div(class='text-wrapper')
         div(class='isso-comment-header' role='meta')
             if bool(comment.website)
-                a(class='author' href='#{comment.website}' rel='nofollow')
+                a(class='author' href=comment.website rel='nofollow')
                     = bool(comment.author) ? comment.author : i18n('comment-anonymous')
             else
                 span(class='author')
                     = bool(comment.author) ? comment.author : i18n('comment-anonymous')
             span(class="spacer") &bull;
-            a(class='permalink' href='#isso-#{comment.id}')
-                time(title='#{humanize(comment.created)}' datetime='#{datetime(comment.created)}')
+            a(class='permalink' href='#isso-' + comment.id)
```

Comment:
There are still usages of `${}` on lines 35-36-37 below in this file

Reply:
See above comment